src/autogen.sbt

```diff
@@ -0,0 +1,415 @@
+// Automatically generated, DO NOT EDIT
+
+val topDir = file(".")
```

Comment:
I don't believe this file is needed, please remove

Reply:
done. thanks